docs/tools/mbed_targets.md

```diff
@@ -148,6 +149,40 @@ The build system errors when you use features outside of this list.
 
 When you use target inheritance, you may alter the values of `features` using `features_add` and `features_remove`. This is similar to the `macros_add` and `macros_remove` mechanism the previous section describes.
 
+#### `config` and `overrides`
+
+_configs_ provide a more flexible way to manage macros for a target. Each configuration has a macro name, as well as a default value and an optional help value. For example, this config flag defines a configuration `lf_clock_src` that defines the source for the low frequency clock on the nRF51 target, with a default value of `NRF_LF_SRC_XTAL`. Compiling sets this value for the macro `MBED_CONF_NORDIC_NRF_LF_CLOCK_SRC`:
+
+```json
+"config": {
+    "lf_clock_src": {
+        "value": "NRF_LF_SRC_XTAL",
+        "macro_name": "MBED_CONF_NORDIC_NRF_LF_CLOCK_SRC"
+    }
+}
+```
+
+_overrides_ allow a child target to change the value of a config. For example, if a child target of the nRF51 uses the internal RC clock instead of the crystal, it can add an override:
```

Comment:
Query: Is this example specific to the nRF51, or does it work for other boards, too?